### PR TITLE
job search

### PR DESCRIPTION
**Pull request by Emm-aan1**
just added my name to submission

### CODE REVIEW

File: Submissions/@emm-aan1.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Ifeanyi Chidoka",
+  "img": "https://avatars.githubusercontent.com/u/114946207?s=400&u=d058aa0a85c614014a0af2b20690e8671acdba73&v=4",
+  "email": "ik.chidoka@gmail.com",
+  "links": {
+    "website": "emm-aan1.github.io/mine",
```
Comment (ProsperousHeart) [inline Submissions/@emm-aan1.json:6]:
```suggestion
    "website": "https://emm-aan1.github.io/mine",
```

File: Submissions/@emm-aan1.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "linkedin": "https://www.linkedin.com/in/ifeanyi-chidoka-58a02614a/",
+    "github": "https://github.com/Emm-aan1"
+  },
+  "jobTitle": "Frontend Developer",
+  "location": {
+    "city": "Warri",
+    "state": "Delta",
+    "country": "Nigeria"
+  }
+}
```
Comment (ProsperousHeart) [inline Submissions/@emm-aan1.json:16]:
```suggestion
}

```

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
aside from suggested changes, please remove the `@` from filename as it needs to match you `Emm-aan1` username

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Looks great except for the filename. Please ensure it matches your username exactly:  `Emm-aan1`

**Comment by ProsperousHeart** [comment]
FYI @Emm-aan1 you shouldn't need to close and open a new PR to complete this :)

**Comment by Emm-aan1** [comment]
I know. But for some reasons unknown to me, I couldn't push the changes so I was a bit confused

**Pull request closed without merging**